engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -141,15 +141,15 @@ object JesBackend {
     val delay = backoff.backoffMillis.millis
 
     if (retries > 0) {
-      f recoverWith {
+      f() recoverWith {
```

Comment:
I suppose I could pull the `f()` into a single spot. I'll leave it up to reviewers.